docs/examples/image_morphology/morphological_operations.jl

```diff
@@ -0,0 +1,73 @@
+# ---
+# title: Convex Hull, Filling, Thinning, ClearBorder
+# author: Ashwani Rathee
+# date: 2021-7-4
+# ---
+
+# In this demonstration, we will be covering morphological operations like
+# convexhull, image thinning, hole filling and border clearning.
+
+using Images, Noise, ImageMorphology
+using HistogramThresholding
+using MosaicViews, ImageDraw
+using TestImages
+
+img_src = testimage("morphology_test_512")
+
+# Thresholding of image
+
+edges1, counts1 = build_histogram(Gray.(img_src), 256)
+t1 = find_threshold(UnimodalRosin(), counts1[1:end], edges1)
+img_input = zeros(Bool, axes(Gray.(img_src)))
+for i in CartesianIndices(img_src)
+    img_input[i] = Gray.(img_src[i]) < t1 ? 0 : 1
+end
+img_input
```

Comment:
Isn't this `binarize(Gray{Bool}, img_src, UnimodalRosin())`?

It currently doesn't work, so `Gray{Bool}.(binarize(img_src, UnimodalRosin()))` (https://github.com/zygmuntszpak/ImageBinarization.jl/issues/83)

Reply:
neither one of those work really

Reply:
Because `morphology_test_512` is `RGBA`, so need to change it to: `RGB.(testimage("morphology_test_512"))`.

The better approach is to update the upstream image in TestImages.

Reply:
More like `binarize(Gray.(img_src), UnimodalRosin()) .> 0.5`
Operations require 512*512 BitMatrix, and not Gray.{Bool}, slight modification works

Reply:
I would need to update both of those 😓